api/agent/drivers/driver.go

```diff
@@ -75,6 +75,9 @@ type Driver interface {
 	// Set image pull retry policy and retriable error checker
 	SetPullImageRetryPolicy(policy common.BackOffConfig, checker RetryErrorChecker) error
 
+	// Get serialized call extensions to select a slot to place the call
```

Comment:
Would you please expand on this comment just a bit further?  In particular, it might be helpful to note that this allows driver implementations to select what change to extension fields warrant creation of a new container, versus reusing an existing one.

Reply:
updated